inference-engine/include/cldnn/cldnn_config.hpp

```diff
@@ -11,47 +11,11 @@
 #pragma once
 
 #include "ie_plugin_config.hpp"
+#include "ie_api.h"
+#include "gpu/gpu_config.hpp"
```

Comment:
maybe we need to remove `gpu/gpu_config.hpp` include from here? In this case users *have* to migrate to new config file to utilize its options

Reply:
I have moved GPU_THROUGHPUT_STREAMS option to gpu_config.hpp, so for backward compatibility it have to be included into cldnn_config.hpp